bigflow/cli.py

```diff
@@ -318,15 +320,41 @@ def _add_build_dags_parser_arguments(parser):
                         type=bf_commons.valid_datetime)
 
 
+def _add_build_image_parser_arguments(parser: argparse.ArgumentParser):
+    parser.add_argument(
+        '--export-image-tar', dest='export_image_tar', action='store_true',
+        help="Export built docker image into .tar file",
+    )
+    parser.add_argument(
+        '--no-export-image-tar', dest='export_image_tar', action='store_false',
+        help="Don't epxort built docker image into .tar file (keep image in local docker registry)",
+    )
+    parser.set_defaults(export_image_tar=None)
+
+    parser.add_argument(
+        '--cache-from-image', dest='cache_from_image',
+        help="Docker images to consider as cache sources",
+    )
+    parser.add_argument(
+        '--cache-from-version', dest='cache_from_version',
+        help="Use previous version of the project as cache source",
+    )
+    _add_dockerrepo_auth_parsers_arguments(parser)
+
+
 def _create_build_dags_parser(subparsers):
     parser = subparsers.add_parser('build-dags',
                                    description='Builds DAG files from local sources to {current_dir}/.dags')
     _add_build_dags_parser_arguments(parser)
 
 
 def _create_build_image_parser(subparsers):
-    subparsers.add_parser('build-image',
-                          description='Builds a docker image from local files.')
+    parser = subparsers.add_parser(
+        'build-image',
+        description='Builds a docker image from local files.',
+    )
+    _add_build_image_parser_arguments(parser)
+    _add_parsers_common_arguments(parser)
```

Comment:
Do we have to add a common parser?

Reply:
For now yest. It contains '-c'/'--config' option, which is used to resolve 'vault' token/key from 'deployment_config.py'

Reply:
👍